scss/modules/_typography-fonts.scss

```diff
@@ -25,3 +25,6 @@ $headingFont: 'Roboto Slab', 'serif';
  * }
  */
 $mainFont: 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:800");
+$headingEventFont: 'Open Sans', 'sans-serif';
```

Comment:
As it is written, it seems that these two lines are meant only for events. But the typography module is loaded in all projects.

If the above is correct, these changes will introduce bloat in every other project but events.